build.zig

```diff
@@ -41,6 +41,7 @@ pub fn build(b: *zbs.Builder) !void {
     const scanner = ScanProtocolsStep.create(b);
     scanner.addSystemProtocol("stable/xdg-shell/xdg-shell.xml");
     scanner.addSystemProtocol("unstable/xdg-output/xdg-output-unstable-v1.xml");
+    scanner.addProtocolPath("protocol/pointer-gestures-unstable-v1.xml");
```

Comment:
This protocol should be available from the system wayland-protocols package
```suggestion
    scanner.addSystemProtocol("unstable/pointer-gestures-unstable-v1.xml");
```